src/libs/ajax/Runtimes.ts

```diff
@@ -34,60 +37,61 @@ export const Runtimes = signal => {
         return fetchLeo(root, _.mergeAll([authOpts(), jsonBody(body), { signal, method: 'POST' }, appIdentifier]))
       },
 
-      update: options => {
+      update: (options): Promise<void> => {
         const body = { ...options, allowStop: true }
         return fetchLeo(root, _.mergeAll([authOpts(), jsonBody(body), { signal, method: 'PATCH' }, appIdentifier]))
       },
 
-      start: () => {
+      start: (): Promise<void> => {
         return fetchLeo(`${root}/start`, _.mergeAll([authOpts(), { signal, method: 'POST' }, appIdentifier]))
       },
 
-      stop: () => {
+      stop: (): Promise<void> => {
         return fetchLeo(`${root}/stop`, _.mergeAll([authOpts(), { signal, method: 'POST' }, appIdentifier]))
       },
 
-      delete: deleteDisk => {
+      delete: (deleteDisk: boolean): Promise<void> => {
         return fetchLeo(`${root}${qs.stringify({ deleteDisk }, { addQueryPrefix: true })}`,
           _.mergeAll([authOpts(), { signal, method: 'DELETE' }, appIdentifier]))
       }
     }
   }
 
-  const v2Func = (workspaceId, name, cloudProvider = 'azure') => {
-    const root = `api/v2/runtimes/${workspaceId}/${cloudProvider}/${name}`
+  const v2Func = (workspaceId: string, name: string, cloudProvider: CloudPlatform = 'AZURE') => {
```

Comment:
nit: should change this cloudProvider to cloudPlatform (since the name isn't being used in an API request)